astro_2.0/features/Notifications/hooks.ts

```diff
@@ -50,3 +62,251 @@ export function useNotificationsSettings(): {
     updateSettings,
   };
 }
+
+export function useNotificationsList(
+  accountDaosIds?: string[],
+  subscribedDaosIds?: string[]
+): {
+  notifications: PaginationResponse<Notification[]> | null;
+  loadMore: () => void;
+  handleRemove: (
+    id: string,
+    {
+      isMuted,
+      isRead,
+      isArchived,
+    }: {
+      isMuted: boolean;
+      isRead: boolean;
+      isArchived: boolean;
+    }
```

Comment:
Looks like this construction repeats pretty often. Can we create some type and reuse it?

Reply:
done